src/diffusers/pipelines/text_to_video_synthesis/pipeline_text_to_video_synth.py

```diff
@@ -0,0 +1,667 @@
+# Copyright 2023 Alibaba DAMO-VILAB and The HuggingFace Team. All rights reserved.
```

Comment:
Add alibaba citation

Reply:
Probably not a big deal, but the pipeline code is arguably mostly HF :)

Reply:
Yeah true, removed it there since there is no code really from alibiba except the tensor2vid which is tiny and whree we left a link and comment